my-app/screens/OTP.js

```diff
@@ -0,0 +1,269 @@
+import React from "react";
+import {
+  AppRegistry,
+  StyleSheet,
+  Image,
+  Button,
+  Text,
+  View,
+  StatusBar,
+  ScrollView,
+  TouchableOpacity,
+  Alert
+} from "react-native";
+import CodeInput from 'react-native-confirmation-code-input';
+import Background from "../custom/Background";
+
+export default class OTP extends React.Component {
+
+  mobileNumber={myMobileNumber:'+91 8888888888'}
```

Comment:
please make it phone number dynamic

Reply:
This is a demo app. I can't add a dynamic number.

Reply:
okay but please make it proper when API is done